packages/clerk-js/src/ui/signIn/SignInAccountSwitcher.test.tsx

```diff
@@ -28,12 +28,7 @@ jest.mock('ui/contexts', () => {
         },
       },
       authConfig: {
-        identification_strategies: [
-          'email_address',
-          'oauth_google',
-          'oauth_facebook',
-        ],
-        first_factors: ['email_address', 'oauth_google', 'oauth_facebook'],
+        singleSessionMode: false,
```

Comment:
❓ Is the `authConfig.singleSessionMode` the only attribute left on the AuthConfig object? Do you think we should merge it with UserSettings?

Reply:
Hi Sokrati, this is the case indeed. We will use `UserSettings` for anything related to the user, like identification requirements, attributes etc. In the future, it's possible that we will introduce a separate `SessionSettings` concept holding information related to the current instance, like `singleSessionMode`.

Once we're done refactoring, we will remove every other property from AuthConfig to prevent misuse. Hope this helps!